safe/gui/tools/peta_jakarta_dialog.py

```diff
@@ -537,3 +528,26 @@ def reject(self):
         # add our own logic here...
 
         super(PetaJakartaDialog, self).reject()
+
+    def download(self, url, output_path):
+        """Download file from url and write to output path.
+
+        :param url: URL of the zip bundle.
+        :type url: str
+
+        :param output_path: Path of output file,
+        :type output_path: str
+        """
+        downloader = FileDownloader(url, output_path)
+        try:
+            result = downloader.download()
+        except IOError as ex:
```

Comment:
ok cool. so, probably I will replace this exception with the original one. Is it fine?